ci-operator/step-registry/acm/fetch-managed-clusters/OWNERS

```diff
@@ -0,0 +1,13 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# Fetched from https://github.com/rhpit/interop-tests root OWNERS
+# If the repo had OWNERS_ALIASES then the aliases were expanded
+# Logins who are not members of 'openshift' organization were filtered out
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- cspi-qe-ocp-lp
+- vboulos
```

Comment:
Yes I've excluded two other user from this OWNERS file as they have no shared ownership over the fetch-managed-clusters ref. I figured if someone where to read the OWNERS file and reach out to them about this ref it would be a waste of their time since they probably couldn't help.

So this is just my attempt of putting only the PQE who have shared ownership in the OWNERS file for specific refs.

Reply:
Okay perfect, I just wanted to verify it wasn't a mistake. There are a lot of OWNERS files to keep track of 🙂